cmd/commands/tx.go

```diff
@@ -12,7 +12,7 @@ import (
 
 	cmn "github.com/tendermint/go-common"
 	client "github.com/tendermint/go-rpc/client"
-	"github.com/tendermint/go-wire"
+	wire "github.com/tendermint/go-wire"
```

Comment:
hmm okay. I use fatih/vim-go too but this doesn't occur in my editor.. (is this new?), aka just tried to compile your code from vim with and without `wire` imports prefix and it works. Maybe I'm missing something, get the logic, curious as to where this comes from however